src/content/support/p1-process/index.md

```diff
@@ -0,0 +1,92 @@
+---
+linkTitle: P1 Incident Process
+title: Critical incident process
+description: How Giant Swarm support team react to P1 incidents.
+weight: 100
+menu:
+  main:
+    parent: overview
+    weight: 20
+user_questions:
+  - What process does Giant Swarm follow in case of critical incidents?
+owner:
+  - https://github.com/orgs/giantswarm/teams/teddyfriends
+last_review_date: 2023-04-25
```

Comment:
```suggestion
description: Critical Incident Process description followed by the Giant Swarm support team when a priority one incident is called.
weight: 100
menu:
  main:
    parent: overview
    weight: 20
user_questions:
  - What process does Giant Swarm follow in case of critical incidents?
owner:
  - https://github.com/orgs/giantswarm/teams/teddyfriends
last_review_date: 2023-04-24
```